### PR TITLE
Added precision about XQuartz in SUMO interface

### PR DESCRIPTION
**Pull request by DavidMansolino**
Fix issue omichel/webots#4167

Link: https://www.cyberbotics.com/doc/automobile/sumo-interface?version=fix-sumo-xquartz-version

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: automobile/sumo-interface.md [hunk 1]
```diff
@@ -57,6 +57,10 @@ field `controllerArgs` in order to customize the behavior of the interface:
 %end
 
 
+> **note** [Mac OSX]:
+On mac OSX, SUMO relies on X11. You need therefore to install [XQuartz](https://www.xquartz.org) (version 2.7.8 or later) for the interface to work.
```
Comment (stefaniapedrazzi) [inline automobile/sumo-interface.md]: I not completely sure about the syntax we use in the documentation but I think should be ``Mac OS X``.

Comment (DavidMansolino) [inline automobile/sumo-interface.md]: I have checked and we use `Mac OS X` everywhere.

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**